### ISSUE TITLE
input value disappears on blur

### ISSUE THREAD

**Issue by dustinfarris** [miguelcobain/ember-paper#411]
It seems this rule in angular/material is biting me: https://github.com/angular/material/blob/v1.0.6/src/components/input/input.scss#L136-L148

To reproduce just start a fresh ember project and install ember-paper master, than add application.hbs:

```hbs
{{paper-input onChange=null}}
```

You'll see that you can type into the input like usual, but when you click away the text disappears (becomes transparent).

I think this is because the class `md-input-has-value` is not being added to the md-input-container

**Comment by dustinfarris** [comment]
the issue is not the css

**Comment by dustinfarris** [comment]
ember-paper is extremely strict DDAU.  you will have to create intermediary components on your own if this breaks down for you.

**Comment by miguelcobain** [comment]
Just for future reference, paper-input behaves just like paper-switch, paper-checkbox or paper-radio.
It sends the action up with upon interaction. It calls the action with the new state.
The component will not assume that the state is changed as the user may not want to do that.
It is up to the user to change the state so that the data flows down to the component.

tl;dr the components *always* reflect the state they are given.
